test-core-cli-full-initialize.ini

```diff
@@ -0,0 +1,51 @@
+[server:main]
+use = config:test-core.ini
+
+[app:main]
+use = config:test-core.ini
+
+ckan.datastore.write_url = postgresql://datastore_write:pass@ckan-postgres/datastore_test
```

Comment:
Do we want to be consistent here with [test-core.ini ](https://github.com/ckan/ckan/blob/master/test-core.ini#L22)file and use localhost instead of ckan-postgres?